projects/angular/src/data/datagrid/datagrid-virtual-scroll.directive.spec.ts

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { ChangeDetectorRef, Component, CUSTOM_ELEMENTS_SCHEMA, OnInit, ViewChild } from '@angular/core';
+import { ComponentFixture, fakeAsync, flush, TestBed, tick } from '@angular/core/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { animationFrameScheduler, BehaviorSubject, Observable } from 'rxjs';
+
+import { ClarityModule } from '../../clr-angular.module';
+import { CustomClrVirtualRowsDirective } from './datagrid-virtual-scroll.direcive';
+import { DATAGRID_SPEC_PROVIDERS } from './helpers.spec';
+
+export interface Column {
+  index: number;
+  name: string;
+}
+
+export interface Row {
+  index: number;
+  expanded: boolean;
+  cells: any[];
```

Comment:
As discussed here could be used some typization instead of any

I think we discussed that example:

`interface cells {
   [key: string]: string;
}`

Reply:
Corrected.